### DISCUSSION TITLE
Must use import to load ES Module (using ky/ky-universal library)

### DISCUSSION THREAD

**Discussion by matveychuk** [vercel/next.js#21253] [Help]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

14.15.4

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**How are you deploying your application?**

localhost

**Describe the Bug**

When trying to use ky/ky-universal with the newly created Next.js app receive such error:
Error: Must use import to load ES Module: /Users/---/node_modules/ky-universal/index.js
require() of ES modules is not supported.
require() of /---/node_modules/ky-universal/index.js from /---/.next/server/pages/index.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename /---/node_modules/ky-universal/index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /---/node_modules/ky-universal/package.json.

Packages version:
Node 14.15.4,
"dependencies": {
    "ky": "^0.26.0",
    "ky-universal": "^0.9.1",
    "next": "10.0.5",
    "react": "17.0.1",
    "react-dom": "17.0.1"
  }
  
  Code extract
```
import ky from "ky-universal";

export default function Home() {
  const fetchData = async () => {
    const res = await ky
      .get("https://jsonplaceholder.typicode.com/todos/1")
      .json();
    setData(res);
    console.log(res);
  };
```
Maintainers of ky state that "This package(ky/ky-universal) is correctly set up for ESM, so Next.js is doing something wrong."

**Expected Behavior**

Possibility to fetch data with ky library

**To Reproduce**

Create new Next.js app, add ky and ky-universal and try to do any HTTP request.

**Comment by oldo** [comment]
Also experiencing the same issue
https://github.com/vercel/next.js/discussions/21252

**Comment by reducio** [comment]
Yes, I have same problem with ky/ky-universal packages.

**Reply by reducio** [reply]
> Yes, I have same problem with ky/ky-universal packages.

https://github.com/vercel/next.js/discussions/21252#discussioncomment-304146

**Comment by saltyshiomix** [comment]
This is a Next.js side problem.
(I sent [the same issue](https://github.com/sindresorhus/ky-universal/issues/28) here.)

In the meantime, we can use `ky@0.25.1` and `ky-universal@0.8.2` which should fix the errors.

Related: https://github.com/sindresorhus/ky/issues/336
